.github/workflows/cleanup-version-branch.yml

```
@@ -0,0 +1,26 @@
+jobs:
+  cleanup:
+    name: Delete release-next branch after merge
+    if: github.event.pull_request.merged == true && startsWith(github.head_ref, 'release-next-v')
+    runs-on: ubuntu-24.04
+    permissions:
+      contents: write
+    steps:
+      - name: Delete branch
+        uses: actions/github-script@f28e40c7f34bde8b3046d885e986cb6290c5673b # v7.1.0
+        with:
+          script: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)